addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -253,3 +253,18 @@ $-o-bg-colors-custom: null;
     background-color: $input-focus-bg;
     border-color: $o-input-hover-border-color;
 }
+
+
+// Inputs fields
+// Creating a different styling for toggle switches with a purpose of switching from A to B rather than ON/OFF
+// ============================================================================
+.o_switch_toggle{
+    .form-check-input{
+        border-color: $o-action !important;
```

Comment:
Is it important really necessary ? 😢

Reply:
No 🙌

It was only due to the fact I wasn't specific enough in my selectors